crypto/rand/rand_unix.c

```diff
@@ -264,7 +253,7 @@ static ssize_t syscall_random(void *buf, size_t buflen)
      * Note: 'buflen' equals the size of the buffer which is used by the
      * get_entropy() callback of the RAND_DRBG. It is roughly bounded by
      *
-     *   2 * RAND_POOL_FACTOR * (RAND_DRBG_STRENGTH / 8) = 2^14
```

Comment:
Look like you mixup some thing in you git commands, and so some recent changes in this code are reverted